media/css/firefox/ios.less

```diff
@@ -76,20 +79,21 @@ a.go {
 // Main hero space
 .section-intro {
     .container {
-        padding: 40px 500px 80px 10px;
+        padding: 60px 500px 80px 10px;
         min-height: 550px;
-        background-position: 570px 0;
+        background-position: 570px 20px;
```

Comment:
If it was a padding or margin I'd agree, but for `background-position` I think it should be a more explicit value.